apps/web/components/booking/BookingListItem.tsx

```diff
@@ -340,6 +338,7 @@ function BookingListItem(booking: BookingItemProps) {
 
       <tr data-testid="booking-item" className="hover:bg-muted group flex flex-col sm:flex-row">
         <td
+          role="a"
           className="hidden align-top ltr:pl-6 rtl:pr-6 sm:table-cell sm:min-w-[12rem]"
           onClick={onClickTableData}>
```

Comment:
ideally we remove the `onClick={onClickTableData}` and replace it with `<Link href={bookingLink} />`